### DISCUSSION TITLE
Visual reactive Programming - Bonsai - NeuroKit Course

### DISCUSSION THREAD

**Discussion by bruno-f-cruz** [bonsai-rx/bonsai#988] [General]
**Hi everyone!**

**I will just leave a copy of the [google groups' post announcing the CAJAL NeuroKit course on Bonsai](https://groups.google.com/g/bonsai-users/c/Xp_PiK7bPTY) here:**

**Hope you see you there!**

-----
Dear Bonsai users,  

The Cajal Programme is organising the 2nd NeuroKit course on Bonsai, called [Visual Reactive Programming - Bonsai](https://cajal-training.org/2022/bonsai1022/), the 24th - 28th of October 2022. We started to run these courses purely online due to the covid pandemic, but we are now also looking for groups of students and TAs who want to join the course as a group and get training at their own Institute/University while connected with the online faculty. 

Course overview
This course will introduce the basics of the [Bonsai programming language](https://bonsai-rx.org/), which allows scientists with no previous programming experience to quickly develop and scale-up experimental rigs, and integrate new open-source hardware and software. By the end of the course, you will be able to:

– create data acquisition and processing pipelines for video and visual stimulation.
– control behavioral task states and run your closed-loop experiments.
– collect data from cameras, microphones, Arduino boards, electrophysiology devices, etc.
– achieve precise synchronization of independent data streams.

 

What is a NeuroKit Course? 

The  NeuroKit courses combine online lectures about advanced neuroscience topics with hands-on experiments. Participants from anywhere receive an experimental box (the kit) containing all the course material to run experimental projects. Participants watch pre-recorded lectures at their own pace in the morning, and attend afternoon sessions with the online faculty from 1-5pm UK time. 

 

Applications

- If you are interested in taking the course online, please fill in this [FORM](https://form.jotform.com/221843075207351).  

- If you want to participate as a satellite/local group of minimum 4 people with your local Teaching Assistant (TA) or our online TAs at your University/Institute using the NeuroKit course material please fill in this[ FORM](https://form.jotform.com/221982450403350). 

For registration fees and more information about the course please read the main [course webpage](https://cajal-training.org/2022/bonsai1022/). 

Bear in mind that there are studentships and honoraria for TAs available, and that the Cajal Programme is highly committed to inclusion and diversity. 

If you have any question, please email [in...@cajal-training.org](http://info@cajal-training.org/) 

Help us to run Bonsai courses around the world, and please spread the news about this course. Thank you!

Best regards,

Elena

**Comment by marta-vp** [comment]
Hi Bruno,
I'm very interested in taking the course but unfortunately I was not selected for this run.
Are there other alternatives? 
Thanks!!
Marta
